spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/cache/CacheProperties.java

```diff
@@ -114,6 +120,24 @@ public Resource resolveConfigLocation(Resource config) {
 		return null;
 	}
 
+	/**
+     * Cache2k specific cache properties.
+     */
+	public static class Cache2k {
+
+		/**
+		* Alternative cache2k cache manager name to use.
```

Comment:
"Alternative" sounds a bit odd here. What is the purpose of this name?

Reply:
Unlike Guava or Caffeine, caches in cache2k are managed. That means caches have a name and live within a cache manager. This allows caches to have external configuration or monitoring exports. When working on a larger code base, it can make sense to separate caches into different managers, e.g. "spring" for the caches used by the Spring cache abstraction, "hibernate" for database caches. The advantage is, besides better overview, that you can apply different default configurations to a group of caches. For example, enable micrometer export for all hibernate caches as well.

When running multiple application containers in parallel, it is also needed to separate the caches into different managers to avoid conflicts.

Putting the manager name in the Spring configuration is basically the same concept as the config resource in JCache. However, a config resource doesn't make sense for cache2k since file based configuration is optional.

I think it is a good idea to put a bit more in the Java doc comment.